cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -50,9 +50,9 @@ current-context: service-account-context
 EOF
   local -r kubeconfig_base64=$(echo "${kubeconfig}" | base64 -w0)
   read -r -d '' secretyaml <<EOF
-apiVersion: v1beta1
+apiVersion: v1beta3
 kind: Secret 
-id: token-${safe_username}
+name: token-${safe_username}
```

Comment:
I took the first 2 bullet points to mean that any existing "name" field moved to ObjectMeta and "id" was renamed to "name". So here "id" changed to "name" and then moved to ObjectMeta?